main.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-undef */
 const mobileMenu = document.getElementById('mobile-menu');
 const body = document.querySelector('.body');
 const menuItemZero = document.querySelector('.mobile-menu-item-0');
```

Comment:
- [ ] Kindly fix `// eslint-disable-next-line no-unused-vars` instead of disableling it, the linters are in palce to help you improve your codding practices and make you a better programmer

Reply:
this is because the function is being call from the HTML file instead of the javascript file. Is there a solution to that?